src/mame/sanritsu/jantotsu.cpp

```diff
@@ -493,7 +493,7 @@ void jantotsu_state::machine_reset()
 void jantotsu_state::jantotsu(machine_config &config)
 {
 	// basic machine hardware
-	Z80(config, m_maincpu, 18.432_MHz_XTAL / 4);
+	Z80(config, m_maincpu, XTAL(18'432'000) / 4);
 	m_maincpu->set_addrmap(AS_PROGRAM, &jantotsu_state::prg_map);
 	m_maincpu->set_addrmap(AS_IO, &jantotsu_state::io_map);
```

Comment:
Please don’t do this.  It’s easier to read using literal suffixes.

Reply:
I'm sorry. I misunderstood the notation policy.